File: 217-ContainsDuplicate/217-ContainsDuplicate.java
// Last updated: 9/14/2025, 12:53:55 PM
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> hs = new HashSet<>();
        for(int n : nums){
            if (hs.contains(n)) return true;
            else hs.add(n);
        }
        return false;
    }
}

File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 9/14/2025, 12:53:52 PM
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        visited = set()
        for num in nums:
            if num in visited: return True
            else: visited.add(num)
        return False